test/controllers/works_controller_test.rb

```diff
@@ -0,0 +1,48 @@
+require "test_helper"
+
+describe WorksController do
```

Comment:
Some negative tests for the WorksController are needed, like deleting a work that already exists, and showing a work that doesn't exist, etc.